.typos.toml

```diff
@@ -1,31 +1,30 @@
-[default.extend-identifiers]
-AnserStyle = "AnserStyle"
-datas = "datas"
-impl_froms = "impl_froms"
-selfs = "selfs"
+[files]
+extend-exclude = [
+    "bench_data/",
+]
+ignore-hidden = false
+
+[default]
+extend-ignore-re = [
+    # ignore string which contains $0, which is used widely in tests
+    ".*\\$0.*",
+    # ignore generated content like `boxed....nner()`, `Defaul...efault`
+    "\\w*\\.{3,4}\\w*",
+    '"flate2"',
+    'IDENT "SER"',
+    "raison d'être",
+]
 
 [default.extend-words]
 anser = "anser"
 ba = "ba"
 fo = "fo"
 ket = "ket"
 makro = "makro"
-raison = "raison"
 trivias = "trivias"
-TOOD = "TOOD"
 
-[default]
-extend-ignore-re = [
-    # ignore string which contains $x (x is a num), which use widely in test
-    ".*\\$\\d.*",
-    # ignore generated content like `boxed....nner()`, `Defaul...efault`
-    "\\w*\\.{3,4}\\w*",
-]
-
-[files]
-extend-exclude = [
-  "*.json",
-  "*.rast",
-  "crates/parser/test_data/lexer/err/*",
```

Comment:
these are auto generated, IMO they should be exclude, since other typos occur here in the furture

Reply:
Reverted.